securedrop/journalist_app/col.py

```diff
@@ -61,18 +62,26 @@ def delete_single(filesystem_id: str) -> werkzeug.Response:
             current_app.logger.error("error deleting collection: %s", e)
             abort(500)
 
-        flash(gettext("{source_name}'s collection deleted.")
-              .format(source_name=source.journalist_designation),
-              "notification")
+        flash(
+          Markup(
+            gettext(
+                    "<b>Success!</b> The account and data for the source {s} has been deleted."
+                   ).format(s=source.journalist_designation)),
+          "success")
+
         return redirect(url_for('main.index'))
 
     @view.route('/process', methods=('POST',))
     def process() -> werkzeug.Response:
         actions = {'download-unread': col_download_unread,
                    'download-all': col_download_all, 'star': col_star,
-                   'un-star': col_un_star, 'delete': col_delete}
+                   'un-star': col_un_star, 'delete': col_delete,
+                   'delete-data': col_delete_data}
         if 'cols_selected' not in request.form:
-            flash(gettext('No collections selected.'), 'error')
+            flash(
+                Markup(
+                    gettext('<b>Nothing Selected</b> You must select one or more items.')),
```

Comment:
Apologies for not catching this sooner, but would it make sense to add punctuation here? 
For example:
> <b>Nothing Selected.</b> You must select one or more items.

(I've added a comment in the content guide for future discussion, if not)

Reply:
this was a specific ask by @ninavizz - if we were gonna add punctuation a colon would make more sense imo.